src/lib/server.ts

```diff
@@ -358,8 +358,14 @@ export class ClientsController {
     this.cleanupLoggingEventForwarder();
   }
 }
-interface ZwavejsServerOptions {
+
+interface HostConfig {
   port: number;
+  host: string;
+}
+
+interface ZwavejsServerOptions {
+  port: number | HostConfig;
```

Comment:
rather than overloading the `port` option, let's just add `host` and `port` to the options and add a command line option to accept a host as input.

If you are unclear how to do this, lmk

Reply:
No idea why I didn't do that 🤷 .